include/bluetooth/gap.h

```diff
@@ -189,6 +190,140 @@ enum {
 	BT_GAP_SCA_0_20 = 7,
 };
 
+/**
+ * @brief Encode 40 least significant bits of 64-bit LE Supported Features into array values
+ *        in little-endian format.
+ *
+ * Helper macro to encode 40 least significant bits of 64-bit LE Supported Features value into
+ * advertising data. The number of bits that are encoded is a number of LE Supported Features
+ * defined by BT 5.3 Core specification.
+ *
+ * Example of how to encode the `0x000000DFF00DF00D` into advertising data.
+ *
+ * @code
+ * BT_DATA_BYTES(BT_DATA_LE_SUPPORTED_FEATURES, BT_LE_SUPP_FEAT_40_ENCODE(0x000000DFF00DF00D))
+ * @endcode
+ *
+ * @param w64 LE Supported Features value (64-bits)
```

Comment:
I'm curious: What is the `w` in `w64`? :)

Reply:
Well I do not know actually. Since the macros are similar to the one in `uuid.h` I decided to follow the convention there:
https://github.com/zephyrproject-rtos/zephyr/blob/ed94862c81674d1b5a835b01c952f06d98dc7d79/include/bluetooth/uuid.h#L174

Reply:
`width` perhaps? :D @rakons or @joerchan  Do you know/remember?

Reply:
I'm not sure but I think it is word. 32-bit word, word-1, word-2 etc.

Probably `feat` would be better than `w` here.

Reply:
@joerchan agree that `feat` would be better. I'll change the code accordingly.